tests/k8s_env/ksp/ksp_test.go

```diff
@@ -1733,6 +1784,53 @@ var _ = Describe("Ksp", func() {
 
 		})
 
+		It("it can audit access to a dir except readonly access is allowed", func() {
+
+			// Test 1: trying to create a file with directory readonly permissions should generate audit events
+			// Apply KubeArmor Policy
+			err := K8sApplyFile("multiubuntu/ksp-ubuntu-4-audit-file-path-readonly.yaml")
+			Expect(err).To(BeNil())
+
+			// Start KubeArmor Logs
+			err = KarmorLogStart("policy", "multiubuntu", "File", ub4)
+			Expect(err).To(BeNil())
+
+			sout, _, err := K8sExecInPod(ub4, "multiubuntu",
+				[]string{"bash", "-c", "touch /dev/shm/new"})
+			Expect(err).To(BeNil())
+			fmt.Printf("OUTPUT: %s\n", sout)
+
+			expect := protobuf.Alert{
+				PolicyName: "ksp-ubuntu-4-audit-file-path-readonly",
+				Severity:   "10",
+				Action:     "Audit",
+				Result:     "Passed",
+			}
+
+			res, err := KarmorGetTargetAlert(5*time.Second, &expect)
+			Expect(err).To(BeNil())
+			Expect(res.Found).To(BeTrue())
+
+			// Test 2: reading the file should result in success without audit events
+			// Start KubeArmor Logs
+			err = KarmorLogStart("system", "multiubuntu", "File", ub4)
+			Expect(err).To(BeNil())
+
+			sout, _, err = K8sExecInPod(ub4, "multiubuntu",
```

Comment:
here too, use assertcommand

Reply:
why here? no matching logic

Reply:
Matching logic could be .* or it could be NotContain (Permission Denied)
Essentially it's asserting whether the command was success or not